src/lib/components/modal/regularModal/style.module.scss

```diff
@@ -70,25 +70,20 @@
 .container {
   position: relative;
 
-  background-color: white;
-
-  border-radius: 8px;
+  display: flex;
+  align-items: center;
 
   max-width: 592px;
-  width: fit-content;
```

Comment:
Glad to see `width: fit-content` gone as it would lead to some very hard to track bugs sometimes, due to the erratic nature of CSS import order with webpack. [Related ticket (with links to related PRs).](https://linear.app/feather-insurance/issue/STO-4283/investigate-why-css-is-erratic-sometimes-fix)

Reply:
That comment made me go check again if this didn't break the stretching width behaviour of the modal but all good! 😄 

And I agree, if it's causing issues it's good we can rid of it, even it's adding an extra div.